freespace2/freespace.cpp

```diff
@@ -6920,8 +6920,8 @@ DCF(pofspew, "Spews POF info without shutting down the game")
 }
 
 // returns:
```

Comment:
Could you write this into doxygen's format? ex:

`/**
 * @brief description here
 *
 * @returns 0 on a clean exit, or
 * @returns 1 on an error
 */`